src/main/java/org/opentest4j/ValueWrapper.java

```diff
@@ -74,6 +82,12 @@ public static ValueWrapper create(Object value) {
 	 * @since 1.2
 	 */
 	public static ValueWrapper create(Object value, String stringRepresentation) {
+		if (value instanceof ValueWrapper) {
+
+			ValueWrapper wrapper = (ValueWrapper) value;
+			return wrapper.stringRepresentation.equals(stringRepresentation) ? wrapper
+					: create(wrapper.value, stringRepresentation);
+		}
 		return (value == null ? nullValueWrapper : new ValueWrapper(value, stringRepresentation));
```

Comment:
I'm wondering whether one would expect the String representation to be used even if `value` is `null`. @leonard84 WDYT?

Reply:
Fair enough. 🙂